src/pages/AddEmployee.jsx

```diff
@@ -40,18 +47,30 @@ function AddEmployee({ open, onClose }) {
     },
     onError: (err) => {
       setToastSeverity("error");
-      setToastMessage(`Employee could not be added due to:  ${err.message}`);
+      setToastMessage(
+        `Employee data could not be saved due to:  ${err.message}`
+      );
       setOpenToast(true);
     },
   });
   function handleSubmit(event) {
     event.preventDefault();
-    const newEmployee = {
-      fullName,
-      email,
-      employeeType,
-    };
-    mutate(newEmployee);
+    if (employee === null) {
+      const newEmployee = {
+        fullName,
+        email,
+        employeeType,
+      };
+      mutate(newEmployee);
+    } else {
+      const newEmployee = {
+        fullName,
+        email,
+        employeeType,
+        employeeId: employee.employeeId,
+      };
+      mutate(newEmployee);
```

Comment:
for clarity/readability I think the newEmployee should have been a more generic term for both add and edit , eg employeeData

Reply:
Just “employee “ is fine

Reply:
There is a already a prop.employee that will conflict with "employee".